build/buildRunTimeImageBases.sh

```diff
@@ -74,7 +74,7 @@ for dockerFile in $dockerFiles; do
 
         acrRuntimeImageTagNameRepo="$ACR_PUBLIC_PREFIX/$getTagName_result"
 
-        docker tag "$localImageTagName" "$acrRuntimeImageTagNameRepo:$tag"
+        docker tag "$localImageTagName" "$acrRuntimeImageTagNameRepo-$tag"
```

Comment:
nit: Could you add a comment having an example here? (it is getting harder to imagine what the new tag would look like)

Reply:
Added comments in this file to (hopefully) shed some more light on the values of individual registry/tag properties